### ISSUE TITLE
The "storage-systems/analysed-controller-statistics" API call is deprecated

### ISSUE THREAD

**Issue by bazmonk** [treydock/eseries_exporter#25]
In newer e5760s, it appears that the `analysed-controller-statistics` API call has been deprecated and is no longer available.  This looks to be how the controller-statistics is grabbing data.

Is this project maintained at all?  There are slightly different versions of this call that could be used instead.  Wanted to check if there's any interest in fixing this before I go attempting to fork this.

Thanks!

**Comment by bazmonk** [comment]
Never mind: I went at it on my own.

**Comment by treydock** [comment]
@bazmonk If you'd like to open a PR I'd be glad to take a look and get it merged and released.  My site no longer has ESeries hardware so I no longer have the ability to test this exporter.

**Comment by bazmonk** [comment]
Thanks... I'll ask my work about that (since it was for work I have to go
through them to release code).

I must admit I know basically no golang, and just learned enough to make it
work.  I had to adjust the exact API call being made, and I had to adjust
how the URL was parsed in the json unmarshalling.  None of the calls in
your code used a `?foo=bar` query string and the code as-is wasn't parsing
it out correctly.

I also don't have a dev machine that can reach our eseries controllers or
web proxy, so I couldn't see the test portions of the make process work,
but it produces the binary so that works for us.

-Baz
